microsoft-edge/webview2/how-to/prerelease-testing.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Prerelease testing using preview channels
+description: How to test your WebView2 app by using the preview channels of Microsoft Edge (Edge Canary, Edge Dev, or Edge Beta) to find issues that will affect your particular WebView2 app, before WebView2 changes reach  the Evergreen WebView2 Runtime.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 04/22/2024
+---
+# Prerelease testing using preview channels
+
+_Prerelease testing_ means testing your WebView2 app by using the preview channels of Microsoft Edge to find issues that will affect your particular WebView2 app, before any changes ship in the Stable Evergreen WebView2 Runtime.  This helps catch any bugs before they degrade the app experience for end users.
+
+Conduct both automated and manual testing:
+* For **automated testing**, do **prerelease testing**, which means testing with a preview version of WebView2 Runtime.  Covered by the present article.
+* For **manual testing**, use **self-hosting**, which means deploying a preview version of WebView2 Runtime with your app, to use daily by your team.  See [Self-host by deploying preview channels](./self-hosting.md).
+
+See also:
+* [Overview of the Microsoft Edge channels](/deployedge/microsoft-edge-channels)
+* [Become a Microsoft Edge Insider](https://www.microsoft.com/edge/download/insider) - download Edge Canary, Edge Dev, or Edge Beta.
```

Comment:
I am not quite sure how relevant this "See also" is in this section. I would suggest we remove it.

Reply:
@victorhuangwq - in many articles in the docset, agreed - intro section of article needs to be focused.  Instead, leverage big See Also section at bottom of article, to keep Intro clean & focused, rather than becoming a nav-away section.